packages/x-components/src/x-modules/identifier-results/store/__tests__/actions.spec.ts

```diff
@@ -53,6 +54,20 @@ describe('testing identifier results module actions', () => {
   });
 
   describe('fetchAndSaveIdentifierResults', () => {
+    const spiedSearchById = jest.spyOn(adapter, 'searchById');
```

Comment:
The `adapter.searchById` is already a jest spy because the adapter here is a mock. You can apply the expect directly to the adapter object